go.mod

```diff
@@ -156,3 +157,5 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
+
+replace github.com/ava-labs/avalanchego => github.com/ava-labs/avalanchego v1.10.12-0.20230929062838-c67537f30776
```

Comment:
Could we directly take this dependency instead of the replace directive?
This can be done eg `go get -u github.com/ava-labs/avalanchego@branch_name` (or commit id)